components/Card/src/Card/CardAction.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { AnchorHTMLAttributes } from 'react';
+import clsx from 'clsx';
+import './card.scss';
+
+export type Link = (props: AnchorHTMLAttributes<HTMLAnchorElement>) => JSX.Element;
+
+export interface CardActionProps extends AnchorHTMLAttributes<HTMLAnchorElement> {
+  Action?: Link;
+}
+
+const DefaultLink = ({ children, ...props }: AnchorHTMLAttributes<HTMLAnchorElement>) => <a {...props}>{children}</a>;
```

Comment:
This section of the component gets a seemingly unintended style whilst focussing the card (blue border on chrome)
<img width="406" alt="image" src="https://github.com/nl-design-system/denhaag/assets/24610692/faadb666-3992-4ede-81b5-13bb9357c426">